ios/RNCPagerViewManager.m

```diff
@@ -8,12 +8,14 @@ @implementation RNCPagerViewManager
 RCT_EXPORT_MODULE(RNCViewPager)
 
 RCT_EXPORT_VIEW_PROPERTY(initialPage, NSInteger)
-RCT_EXPORT_VIEW_PROPERTY(orientation, NSString)
+RCT_EXPORT_VIEW_PROPERTY(pageMargin, NSInteger)
 
+RCT_EXPORT_VIEW_PROPERTY(orientation, UIPageViewControllerNavigationOrientation)
```

Comment:
Will this work this way? We probably need RCTConvert for this, what determines the orientation conversion to `UIPageViewControllerNavigationOrientation`

Reply:
to be addressed in followup PR